build_tools/benchmarks/suites/iree_tflite_import_template.cmake

```diff
@@ -4,3 +4,5 @@ iree_import_tflite_model(
   SOURCE "$__SOURCE_MODEL_PATH"
   OUTPUT_MLIR_FILE "$__OUTPUT_PATH"
 )
+# Mark dependency so users can import models without compiling them.
```

Comment:
This dependency rule is written twice in `iree_tf_import_template.cmake` and `iree_tflite_import_template.cmake`, which can be avoided if we switch to bazel-cmake builder pattern. However, I haven't figured out how to share the common builder code with `bazel_to_cmake_converter.py` (hit #10065 again...)

I'm actually not sure if it's right to share code between the benchmark infra and other build_tools, as we have discussed to move benchmark code into a separate repo...

Reply:
Looks like we have the solution for #10065, which gives us the freedom to share code. If we are okay to share code with bazel_to_cmake, I can start refactoring in the follow-up changes.

Reply:
Filed #10415 as the follow-up.

Reply:
This isn't the worst repetition :-)